lambdas/process-candidate-identity/src/main/java/uk/gov/di/ipv/core/processcandidateidentity/service/StoreIdentityService.java

```diff
@@ -56,26 +57,32 @@ public void storeIdentity(
             SharedAuditEventParameters auditEventParameters)
             throws EvcsServiceException {
 
+        HttpResponse<String> httpResponse = null;
         if (identityType == CandidateIdentityType.PENDING) {
             evcsService.storePendingIdentity(userId, sessionCredentials, evcsVcs, achievedVot);
         } else {
-            evcsService.storeCompletedIdentity(
-                    userId, sessionCredentials, evcsVcs, strongestMatchedVot, achievedVot);
+            httpResponse =
+                    evcsService.storeCompletedIdentity(
+                            userId, sessionCredentials, evcsVcs, strongestMatchedVot, achievedVot);
         }
 
         LOGGER.info(LogHelper.buildLogMessage("Identity successfully stored"));
 
-        sendIdentityStoredEvent(strongestMatchedVot, identityType, auditEventParameters);
+        sendIdentityStoredEvent(
+                strongestMatchedVot, identityType, auditEventParameters, httpResponse);
     }
 
     private void sendIdentityStoredEvent(
             VotMatchingResult.VotAndProfile strongestMatchedVot,
             CandidateIdentityType identityType,
-            SharedAuditEventParameters auditEventParameters) {
+            SharedAuditEventParameters auditEventParameters,
+            HttpResponse<String> httpResponse) {
 
         boolean sisRecordCreated =
                 configService.enabled(CoreFeatureFlag.STORED_IDENTITY_SERVICE)
-                        && identityType != CandidateIdentityType.PENDING;
+                        && identityType != CandidateIdentityType.PENDING
+                        && !Objects.isNull(httpResponse)
+                        && (httpResponse.statusCode() >= 200 && httpResponse.statusCode() < 300);
```

Comment:
It would be good to be as specific as possible with this - I understand we should get a `202` response